java/client/src/main/java/glide/api/models/commands/json/JsonGetOptions.java

```diff
@@ -0,0 +1,56 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands.json;
+
+import glide.api.commands.servermodules.GlideJson;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Builder;
+import lombok.Getter;
+
+/** Additional parameters for {@link GlideJson#get} command. */
+@Getter
+@Builder
+public final class JsonGetOptions {
+    /** ValKey API string to designate INDENT */
+    public static final String INDENT_VALKEY_API = "INDENT";
+
+    /** ValKey API string to designate NEWLINE */
+    public static final String NEWLINE_VALKEY_API = "NEWLINE";
+
+    /** ValKey API string to designate SPACE */
+    public static final String SPACE_VALKEY_API = "SPACE";
+
+    /** Sets an indentation string for nested levels. Defaults to null. */
+    @Builder.Default private String indent = null;
+
+    /** Sets a string that's printed at the end of each line. Defaults to null. */
+    @Builder.Default private String newline = null;
+
+    /** Sets a string that's put between a key and a value. Defaults to null. */
+    @Builder.Default private String space = null;
+
```

Comment:
TBD String vs GlideString

Reply:
Good question. Do we allow user to use non-UTF8/binary data as INDENT/NEWLINE/SPACE?

Reply:
We should add a `JsonGetOptionsBinary` class to support GlideString. 
I don't think anybody will use it, but Redis supports it and we must too.

Reply:
added `JsonGetOptionsBinary`